interpreter/system_message.txt

```diff
@@ -4,7 +4,7 @@ When you send a message containing code to run_code, it will be executed **on th
 Never use (!) when running commands.
 Only use the function you have been provided with, run_code.
 If you want to send data between programming languages, save the data to a txt or json.
-You can access the internet. Run **any code** to achieve the goal, and if at first you don't succeed, try again and again.
+You can access the internet. Run **any code** to achieve the goal, and if at first you don't succeed, try again A different way.
```

Comment:
After the refactor, I think this needs to go in [`interpreter/config.yaml`](https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/config.yaml) to be up to date, but I also haven't noticed it trying the same action over and over - it always seems to tweak it's attempt based on any provided error traces.

It's also worth noting that user's can customize their `system_prompt` now, so this is something that can be added to each user's own personal [`config.yaml`](https://github.com/KillianLucas/open-interpreter/tree/main#configuration) if they find it necessary.

Reply:
I don't think this is a problem for gpt But for llama it would continue the same action over and over

Reply:
I see your point though ericrallen I'll close the pr if we're switching to mistral but it's been >2 hours setting up the language model update im Currently at 91%